go/client/cmd_status.go

```diff
@@ -220,6 +221,7 @@ func (c *CmdStatus) load() (*fstatus, error) {
 	status.LocalDbStats = extStatus.LocalDbStats
 	status.LocalChatDbStats = extStatus.LocalChatDbStats
 	status.CacheDirSizeInfo = extStatus.CacheDirSizeInfo
+	status.UIRouterMapping = extStatus.UiRouterMapping
```

Comment:
you should log this value as well as the new info in `ClientStatus`, the `status` command should have an output that is similar to the result in the log send

Reply:
Will do, I wasn't sure about it and assumed I'd get an answer in the review :)